runtime/src/startup.rs

```diff
@@ -21,7 +22,11 @@ macro_rules! set_main {
     {$name:ident} => {
         #[no_mangle]
         fn libtock_unsafe_main() -> ! {
-            $name()
+            use libtock_platform;
```

Comment:
`use libtock_platform;` seems like a leftover from a previous iteration of this code -- is it necessary?